docs/layers/grid-layer.md

```diff
@@ -229,6 +249,27 @@ By default `getElevationValue` returns the length of the points array.
 Note: grid layer compares whether `getElevationValue` has changed to recalculate the value for each cell for its elevation.
 You should pass in the function defined outside the render function so it doesn't create a new function on every rendering pass.
 
+
+##### `getElevationWeight` (Function, optional) ![transition-enabled](https://img.shields.io/badge/transition-enabled-green.svg?style=flat-square")
+
+* Default: `point => 1`
+
+`getElevationWeight` is the accessor function to get the weight of a point used to calcuate the elevation value for a cell.
+It takes the data prop array element and returns the weight, for example, to use `SPACE` field, `getElevationWeight` should be set to `point => point.SPACES`.
```

Comment:
It would be nice to provide examples as the combination of these two props to showcase what they do, see comment on `getColorWeight`

Reply:
added examples